content/en/community/hackathons/usoc23/overview/index.md

```diff
@@ -0,0 +1,80 @@
+  We present the setup. The students will use the scripts to run and test all the applications available.
+  Expected time: 100 minutes
+--- 
+
+## Reminders
+
+### Setup
+
+{{< readfile file="/community/hackathons/sessions/setup/content/Setup.md" markdown="true" >}}
+
+### Support Files
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Required-Tools-and-Resources_Support-Files.md" markdown="true" >}}
+
+### linuxu and KVM
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_02.-linuxu-and-KVM.md" markdown="true" >}}
+
+## Overview
+
+### Unikraft Core
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_03.-Unikraft-Core.md" markdown="true" >}}
+
+### Configuring Unikraft - Config.uk
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_05.-Configuring-Unikraft---Config.uk.md" markdown="true" >}}
+
+### The Build System - basics
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Overview_06.-The-Build-System---basics.md" markdown="true" >}}
+
+## Work items
+
+### 01. Tutorial / Reminder: Building and Running Unikraft
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-Tutorial---Reminder:-Building-and-Running-Unikraft.md" markdown="true" >}}
+
+#### Set Up
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-Set-Up.md" markdown="true" >}}
+
+#### linuxu, x86_64
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-linuxu-x86_64.md" markdown="true" >}}
+
+#### KVM, x86_64
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-KVM-x86_64.md" markdown="true" >}}
+
+#### KVM, ARM
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_01.-KVM-ARM.md" markdown="true" >}}
+
+### 02. Tutorial: Make It Speak
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_02.-Tutorial:-Make-It-Speak.md" markdown="true" >}}
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_02.-KVM-x86_64.md" markdown="true" >}}
+
+### 03. Tutorial: Adding Filesystems to an Application
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_06.-Tutorial---Reminder:-Adding-Filesystems-to-an-Application.md" markdown="true" >}}
+
+### 04. Tutorial: Give the User a Choice
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_07.-Tutorial:-Give-the-User-a-Choice.md" markdown="true" >}}
+
+### 05. Tutorial: Arguments from Command Line
+
+{{< readfile file="/community/hackathons/sessions/behind-scenes/content/Work-Items_08.-Tutorial:-Arguments-from-Command-Line.md" markdown="true" >}}
+
+### 06. Echo-back Server
+
+{{< readfile file="/community/hackathons/sessions/baby-steps/content/Practical-Work_01.-Echo-back-Server.md" markdown="true" >}}
```

Comment:
You need to add symbolic links to the `work/` and `sol/` directories where the `Echo-back` server is.